plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -114,6 +114,12 @@ public void testCacheRefreshOnRoleGrantAndRevoke(List<String> grantRoleStatement
                 .hasMessageContaining("Access Denied");
     }
 
+    @Test
+    public void testFlushHiveMetastoreCacheProcedureCallable()
+    {
+        queryRunner.execute("CALL system.flush_hive_metastore_cache()");
```

Comment:
Verify cache has been flushed.
You can do so by
- enabling caching,
- reading a table via Trino
- changing it via Hive in a way that will break the cache
- read from Trino fails
- flush
- read from Trino succeeds

Reply:
Ok, thank's for that. I was thinking on how to do it properly. But wanted to get first round of feedback first.

Reply:
Implemented above test scenarion.